src/rgw/rgw_sal_motr.cc

```diff
@@ -1345,14 +1372,75 @@ bool MotrObject::is_expired() {
   return false;
 }
 
+std::string base62_encode(uint64_t value, size_t pad) 
```

Comment:
code from line 1375 - 1442 is covered in https://github.com/Seagate/cortx-rgw/pull/195 already, so am avoiding going through it again. I believe you have kept this for testing, you can add comment for same accordingly, so can safely remove also after PR #195 is merged

Reply:
Rebased with https://github.com/Seagate/cortx-rgw/pull/195 and removed the duplicate code.